### PR TITLE
Created problem_58 in project_euler

### PR DESCRIPTION
**Pull request by ashujack**
### **Describe your change:**

* [x]  Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [ ] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [ ] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [ ] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by spamegg1** [comment]
`pre-commit`:
>  if __name__ == "__main__":
    print( int((2*i+1)**0.5))
+    print(int((2 * i + 1) ** 0.5))
Error: Process completed with exit code 1.

I think it requires an empty line at the end of the file.

**Comment by ashujack** [comment]
I have added an extra line at end of file still showing same error.
How am I supposed to add it?

File: project_euler/problem_58/sol1.py [hunk 1]
```diff
@@ -25,21 +24,24 @@ def solution():
...
-    return int((2*i+1)**0.5)
+    return int((2 * i + 1) ** 0.5)
 
 
 if __name__ == "__main__":
     from doctest import testmod
+
     testmod()
     print(solution())
+
```
Comment (ashujack) [inline project_euler/problem_58/sol1.py]: I have added an extra line at end of the file still showing the same error.
How am I supposed to add it?

**Review by ashujack — COMMENTED** [review]
I have added an extra line at end of the file still showing the same error.
How am I supposed to add it?

**Comment by sarthaka1310** [comment]
Did you run [black](https://github.com/psf/black)?

**Comment by ashujack** [comment]
> Did you run [black](https://github.com/psf/black)?

No

**Comment by ashujack** [comment]
Thanks, I have forgot to run it.

**Comment by dhruvmanila** [comment]
Only one algorithm in one PR, please.

**Review by dhruvmanila — COMMENTED** [review]
### Announcement:

@ashujack

*This message is sent to all pull requests which are **OPEN, NOT MERGED, not containing 'wontfix' label and contains 'Project Euler' or 'Project' or 'Euler' (case doesn't matter) in their title**. If this message doesn't apply to your pull request, please ignore this.*

### Message:

This is to notify all the users submitting a pull request for the Project Euler solution that the [README.md](https://github.com/TheAlgorithms/Python/blob/master/project_euler/README.md) file has been updated with all the style requirements necessary for the directory and solution files. ***Please read through it and make all the necessary changes required to your submission.***

**Pull request closed without merging**